crates/bevy_window/src/window.rs

```diff
@@ -146,6 +146,10 @@ pub struct Window {
     /// You should also set the window `composite_alpha_mode` to `CompositeAlphaMode::PostMultiplied`.
     pub transparent: bool,
     /// Get/set whether the window is focused.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - Android / iOS / X11 / Wayland: Can't be spawned unfocused.
```

Comment:
is it just at spawn time that it's not possible? can the window be set as unfocused?

Reply:
Thanks I didn't think about checking that.

Setting the window focused/unfocused through code is not possible for "iOS / Android / Web / Wayland" so a different subset of platforms. I'll commit that later.

Reply:
I clarified that in this commit: https://github.com/bevyengine/bevy/pull/9208/commits/1e61bb6663522a08a30c461c864958fbed9fbc87

Note that I think (didn't test to verify) that if it is set to false, or set to true on an unsupported platform, it will just be desynced. I can try to fix that in this PR but I think it would be better in a follow-up PR.